ci/miri.sh

```diff
@@ -3,37 +3,40 @@ set -euxo pipefail
 IFS=$'\n\t'
 cd "$(dirname "$0")"/..
 
+# We need 'ts' for the per-line timing
+sudo apt-get -y install moreutils
+echo
```

Comment:
Ah, I didn't know about this flag.
However (a) it only works with isolation disabled (since for regular tests it actually measures the time *inside* Miri), (b) it is harder to read without colors and libtest output in Miri is never colored for $REASONS, (c) the colors might not be appropriate for Miri; 1.3s seems still pretty good to me. ;)

Up to you, I am fine with removing the `ts` thing again. You have isolation disabled anyway.